about.html

```diff
@@ -12,53 +12,106 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
+<div class = "container">
+    <div class="navbar">
+                
+    <img src="C:\Users\Andrew\Desktop\User-Interface\Sprint-Challenge--User-Interface\img\lambda-black.png" alt="strategy" class="logo">
```

Comment:
This is incorrect image source syntax. When writing the source path for images, **never** set this to a local directory on your machine. Rather, set the image source in relation to where it exists compared to your main directory (usually the one containing index.html). Instead of
`src="C:\Users\Andrew\Desktop\User-Interface\Sprint-Challenge--User-Interface\img\lambda-black.png"`
we would instead want to see
`src="/img/lambda-black.png"`